sdk/digitaltwins/test-resources.bicep

```diff
@@ -0,0 +1,109 @@
+param testApplicationOid string {
+    metadata: {
+        description: 'The client OID to grant access to test resources.'
+    }
+}
+
+param baseName string {
+    default : resourceGroup().name
+    minLength: 6
+    maxLength: 50
+    metadata: {
+        description: 'The base resource name.'
+    }
+}
+
+param location string {
+    default: resourceGroup().location
+    metadata: {
+        description: 'The location of the resource. By default, this is the same as the resource group.'
+    }
+} 
+
+var roleDefinitionId = '/subscriptions/${subscription().subscriptionId}/providers/Microsoft.Authorization/roleDefinitions/8e3af657-a8ff-443c-a75c-2fe8c4bcb635'
+
+resource roleAssignment 'Microsoft.Authorization/roleAssignments@2018-09-01-preview' = {
+    name: guid(resourceGroup().id)
+    properties: {
+        roleDefinitionId: roleDefinitionId
+        principalId: testApplicationOid
```

Comment:
you can directly refer to parameters instead of the lengthy [parameters("testApplicationOid")]